File: evaluation/src/main/java/org/arnhold/evaluation/openaire/OpenAireService.java
package org.arnhold.evaluation.openaire;

import org.apache.jena.rdf.model.Model;

public interface OpenAireService {
    String test(Model madmp);

    void getPublications(Model madmp);
    String getResearchData(Model madmp);
    void getResearchSoftware(Model madmp);
    void getResearchProducts(Model madmp);
}

File: case-study/src/main/kotlin/org/arnhold/casestudy/context/openaire/OpenAireService.kt
package org.arnhold.casestudy.context.openaire

import org.apache.jena.rdf.model.Model

interface OpenAireService {
    fun test(madmp: Model): String

    fun getPublications(madmp: Model)
    fun getResearchData(madmp: Model): String
    fun getResearchSoftware(madmp: Model)
    fun getResearchProducts(madmp: Model)
}
